Makefile

```diff
@@ -46,7 +46,12 @@ ifeq ($(PLUGIN), 1)
 	BUILD_TAGS += with_plugin
 endif
 
-LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(shell git describe --tags --dirty --always)"
+RELEASE_VERSION ?= $(shell git describe --tags --dirty --always)
+ifeq ($(RUN_CI), 1)
+	RELEASE_VERSION := None
```

Comment:
Why do we need to set it to `None`?

Reply:
there is no tag in the ci environment, we can not pass https://github.com/tikv/pd/blob/bfc988bb37a738b3bba2e85903d304af5d8532d6/server/util.go#L39-L41  whithout tag :(